include/boost/geometry/strategies/spherical/distance_cross_track.hpp

```diff
@@ -508,22 +549,22 @@ template
 class cross_track
 {
 public :
-    typedef within::spherical_point_point equals_point_point_strategy_type;
+    using equals_point_point_strategy_type = within::spherical_point_point;
 
-    typedef intersection::spherical_segments
+    using relate_segment_segment_strategy_type = intersection::spherical_segments
         <
             CalculationType
-        > relate_segment_segment_strategy_type;
+        >;
 
     static inline relate_segment_segment_strategy_type get_relate_segment_segment_strategy()
     {
         return relate_segment_segment_strategy_type();
     }
 
-    typedef within::spherical_winding
+    using point_in_geometry_strategy_type = within::spherical_winding
         <
             void, void, CalculationType
-        > point_in_geometry_strategy_type;
+        >;
```

Comment:
These are probably not used at all after the introduction of umbrella strategies.